sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -301,10 +301,12 @@ def send(self, message):
         :param message: The ServiceBus message to be sent.
         :type message: ~azure.servicebus.Message or ~azure.servicebus.BatchMessage or list[~azure.servicebus.Message]
         :rtype: None
-        :raises: :class: ~azure.servicebus.exceptions.MessageSendFailed if the message fails to
-         send
+        :raises:
                 :class: ~azure.servicebus.exceptions.OperationTimeoutError if sending times out.
-                :class: `ValueError` if list of messages is provided and cannot fit in a batch.
+                :class: ~azure.servicebus.exceptions.MessageContentTooLarge if the size of the message is over
+                  service bus frame size limit.
+                :class: ~azure.servicebus.exceptions.MessageSendFailed if the message fails to send
+                :class: ~azure.servicebus.exceptions.ServiceBusError. The parent class of above errors.
```

Comment:
Does this get thrown separately from the above two?  Otherwise I'd feel odd noting it as well, instead of inline as part of the inheritance chain.

Reply:
Yes, there are errors other than the above errors. I changed it to
:class: ~azure.servicebus.exceptions.ServiceBusError when other errors happens.
                 It's also the top-level root class of above errors.

Reply:
If there are other failure cases that throw a ServiceBusError, I'd give an example of that other than just saying "the parent class" since reading it now I was confused and thought it was only being documented AS the parent for the above, not because it represents unique failure cases.